modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import {Text} from '@clayui/core';
+import {ClayCheckbox} from '@clayui/form';
+import {createResourceURL, fetch, openToast} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import './ScriptManagementContainer.scss';
+
+interface ScriptManagementContainerProps {
+	allowScriptContentBeExecutedOrIncluded: boolean;
+	baseResourceURL: string;
+}
+
+export default function ScriptManagementContainer({
+	allowScriptContentBeExecutedOrIncluded,
+	baseResourceURL,
+}: ScriptManagementContainerProps) {
+	const [allowScriptContent, setAllowScriptContent] = useState(
+		allowScriptContentBeExecutedOrIncluded
+	);
+
+	const handleSaveSystemConfiguration = async () => {
+		const response = await fetch(
+			createResourceURL(baseResourceURL, {
+				allowScriptContentBeExecutedOrIncluded: allowScriptContent,
+				p_p_resource_id:
+					'/system_settings/edit_script_management_configuration',
+			}).toString()
+		);
+
+		openToast({
+			message: response.ok
+				? Liferay.Language.get('your-request-completed-successfully')
+				: Liferay.Language.get('an-error-occurred'),
+			type: response.ok ? 'success' : 'danger',
+		});
+	};
+
+	return (
+		<div className="lfr__script-management-container">
+			<Text as="span" size={7} weight="bolder">
+				{Liferay.Language.get('script-management')}
+			</Text>
+
+			<ClayAlert
+				displayType="info"
+				title={`${Liferay.Language.get('alert-helper-info')}:`}
+			>
+				{Liferay.Language.get(
+					'this-configuration-is-not-saved-yet.-the-values-shown-are-the-default'
+				)}
+			</ClayAlert>
+
+			<div className="lfr__script-management-checkbox-container">
+				<div className="lfr__script-management-checkbox-label-container">
+					<ClayCheckbox
+						aria-label={Liferay.Language.get(
+							'allow-instance-admin-to-create-and-execute-code-in-liferay-dxp'
+						)}
+						checked={allowScriptContent}
+						onChange={() =>
+							setAllowScriptContent(!allowScriptContent)
+						}
+					/>
+
+					<Text weight="semi-bold">
+						{Liferay.Language.get(
+							'allow-instance-admin-to-create-and-execute-code-in-liferay-dxp'
+						)}
+					</Text>
```

Comment:
Hi @danvb112, I noticed that semi-bold is not the style applied to this text. But it looks like the pattern applies to the other modules. Could you confirm with Julia if this was a mistake or a design decision, see:

![screenshot-1709920177](https://github.com/liferay-objects/liferay-portal/assets/61854510/bac53094-7fae-48c0-bbda-3f0da6fbfb56)

Reply:
Hey @igor-franca, I spoke to Júlia and she confirmed that it would be better to use the checkbox's default label without the semi-bold, I changed the code, thank you!